api/jwt/jwt.go

```diff
@@ -53,9 +61,10 @@ func NewService(userSessionDuration string) (*Service, error) {
 func (service *Service) GenerateToken(data *portainer.TokenData) (string, error) {
 	expireToken := time.Now().Add(service.userSessionTimeout).Unix()
 	cl := claims{
-		UserID:   int(data.ID),
-		Username: data.Username,
-		Role:     int(data.Role),
+		UserID:     int(data.ID),
```

Comment:
I fail to see why all these fields are filled. I don't find references that actually use these fields? What do I miss here?

Reply:
The claims are the information payload stored in the JWT session token for Portainer. They are converted into a jwt token on line 72 `token := jwt.NewWithClaims(jwt.SigningMethodHS256, cl)`